mantle/kola/tests/misc/network.go

```diff
@@ -560,172 +560,197 @@ func setupBondWithDhcpTest(c cluster.TestCluster, primaryMac, secondaryMac, prim
 	addKernelArgs(c, m, []string{fmt.Sprintf("macAddressList=%s,%s", primaryMac, secondaryMac)})
 }
 
-// NetworkAdditionalNics verifies that additional NICs are created on the node
-func NetworkAdditionalNics(c cluster.TestCluster) {
-	primaryMac := "52:55:00:d1:56:00"
-	secondaryMac := "52:55:00:d1:56:01"
-	ovsBridgeInterface := "ovs-brcnv-iface"
+func simulateNewInstallation(c cluster.TestCluster, m platform.Machine, macConnectionsToDelete []string) error {
+	macConnectionMap, err := getMacConnectionMap(c, m)
+	if err != nil {
+		return fmt.Errorf("failed to get macConnectionMap: %v", err)
+	}
+	macInterfaceMap, err := getMacInterfaceMap(c, m)
+	if err != nil {
+		return fmt.Errorf("failed to get macInterfaceMap: %v", err)
+	}
+	flushInterfaceIpByMac(c, m, macInterfaceMap, macConnectionsToDelete)
+	removeConnectionsByMac(c, m, macConnectionMap, macConnectionsToDelete)
 
-	setupMultipleNetworkTest(c, primaryMac, secondaryMac)
+	err = m.Reboot()
+	if err != nil {
+		return fmt.Errorf("failed to reboot the machine: %v", err)
+	}
+	return nil
+}
 
-	m := c.Machines()[0]
-	checkExpectedMAC(c, m, ovsBridgeInterface, primaryMac)
+func flushInterfaceIpByMac(c cluster.TestCluster, m platform.Machine, macInterfaceMap map[string]string, macsList []string) {
+	for _, mac := range macsList {
+		InterfaceToFlush := macInterfaceMap[mac]
+		c.MustSSH(m, fmt.Sprintf("sudo ip addr flush dev '%s'", InterfaceToFlush))
+	}
+}
+
+func removeConnectionsByMac(c cluster.TestCluster, m platform.Machine, macConnectionMap map[string]string, macsList []string) {
+	for _, mac := range macsList {
+		connectionToDelete := macConnectionMap[mac]
+		c.MustSSH(m, fmt.Sprintf("sudo nmcli con del '%s'", connectionToDelete))
+	}
 }
 
-func checkExpectedMAC(c cluster.TestCluster, m platform.Machine, interfaceName, expectedMac string) {
-	interfaceMACAddress, err := getInterfaceMAC(c, m, interfaceName)
+func simulateNodeReboot(c cluster.TestCluster, m platform.Machine) error {
```

Comment:
That seems like a funny name for a function that actually does reboot the node. :)

Reply:
you're right. removed :)